app/domain/errors.rb

```diff
@@ -411,8 +411,31 @@ module AuthnJwt
 
       MissingToken = ::Util::TrackableErrorClass.new(
         msg: "Token is empty or not found.",
-        code: "CONJ00077E"
+        code: "CONJ00177E"
       )
+
+      InvalidUriConfiguration = ::Util::TrackableErrorClass.new(
+        msg: "Signing key uri authenticator configuration is invalid. It should configured as authenticator variables: " \
+              "one of the following: '{0-resource-name}','{1-resource-name}'",
+        code: "CONJ00178E"
+      )
+
+      InvalidUriConfiguration = ::Util::TrackableErrorClass.new(
+        msg: "Signing key uri authenticator configuration is invalid. It should configured as authenticator variables: " \
+              "one of the following: '{0-resource-name}','{1-resource-name}'",
+        code: "CONJ00179E"
+      )
```

Comment:
think this is too much details for an error msg
maybe we should keep the first sentence and add a troubleshooting section with the error code, like we are doing in GCP authn doc page

@shulifink WDYT?

Anyway for the meanwhile, I've added a troubleshooting section to the confluence doc file

Reply:
@hilagross  - let's discuss

Reply:
@Avi-Eyal i will fix this in the log. Can you update the troubleshooting page according to it.

Reply:
Shortend message to 
"Signing key URI authenticator configuration is invalid"